packages/playground/website/public/button.php

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+<?php
+// Render <a href> that looks like button linking to the current URL except for the start_button parameter
+$url = $_SERVER['REQUEST_URI'];
+$url = preg_replace('/&?start_button=[^&]/', '', $url);
```

Comment:
I know it's small, but maybe we first split the query string from the former part and then split by query arg?

```php
function remove_start_button( $url ) {
	list( $scheme_host_path, $query_hash ) = explode( '?', $url );
	if ( null === $query_hash ) {
		return $url;
	}

	list( $query, $hash ) = explode( $query_hash, '#' );
	if ( empty( $query ) ) {
		return $url;
	}

	$hash = empty( $hash ) ? '' : "#{$hash}";

	$query = preg_replace( '~&?start_button=[^&]*~', '', $query );

	return "{$scheme_host_path}?{$query}{$hash}";
}
```